mne/datasets/utils.py

```diff
@@ -59,6 +59,25 @@
 """
 
 
+_license_text = """
```

Comment:
it's here because the bulk of the download happens here, including asking the user to agree to the license. The brainstorm files don't have much in them ...